src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './_NoOccasions.scss';
+import { NO_OCCASION_TYPE_MESSAGE } from '../../../../constants/common/commonConstants';
+import Container from '../../../common/Typography/Container';
+import IconCake from '../../../common/Typography/Icons/IconCake';
+import IconRing from '../../../common/Typography/Icons/IconRing';
+
+const OccasionsType = (props) => {
+  return (
+    <Container className="no-occasions-container">
+      <Container className={`no-occasions__icon ${props.iconClass}`}>
+        {props.icon}
+      </Container>
+      <Container className="no-occasions__message">
+        {props.message}
+      </Container>
+    </Container>
+  );
+};
+
+const NoOccasion = (props) => {
+  const { members = [], occasionType } = props;
+  let Occasion;
+  if (members.length === 0) {
+    if (occasionType === 'Birthday') {
+      Occasion = <OccasionsType icon={<IconCake />} message={NO_OCCASION_TYPE_MESSAGE.BIRTHDAY_MESSAGE} iconClass="birthday-icon" />
+    } else if (occasionType === 'Anniversary') {
+      Occasion = <OccasionsType icon={<IconRing />} message={NO_OCCASION_TYPE_MESSAGE.ANNIVERSARY_MESSAGE} iconClass="anniversary-icon" />
+    }
+  }
+  return (
+    <Container>
```

Comment:
It will always return an empty Container if no occasion found.

```suggestion
if (!members.length) {
 return <Container> { renderNoOccasionContent( ) }</Container>
}
 return null;
```

Reply:
done

Reply:
??

Reply:
done